src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={classNames({ 'is-active': selectedTab.id === tab.id })}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tab.id}`}
+              onClick={() => {
+                if (selectedTab.id === tab.id) {
+                  return;
+                }
+
+                onTabSelected(tab);
+              }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)
If you need `tab` here you can create this function before tag li. For example move code from 8-26 line to common component and there create function and use it for each tab